src/utils/currency.tsx

```diff
@@ -0,0 +1,64 @@
+
+import React, { useState, createContext, useContext, useEffect } from 'react'
+import BandChain from '@bandprotocol/bandchain.js'
+
+
+const CurrencyContext = React.createContext<null | any>(null);
+const bandchain = new BandChain("http://poa-api.bandchain.org")
```

Comment:
is this a reliable source of fx data?

Reply:
Yes. This data is from Band Protocol's decentralized oracle be on our proof-of-authority mainnet 

https://bandprotocol.com
https://guanyu-poa.cosmoscan.io